client/ctclient/ctclient.go

```diff
@@ -39,6 +39,7 @@ import (
 
 var (
 	logURI    = flag.String("log_uri", "http://ct.googleapis.com/aviator", "CT log base URI")
```

Comment:
Unrelated to this PR but is there a reason we use aviator here?

Reply:
Just historical, I think.  Change it to Pilot or Rocketeer?

Reply:
Maybe. Aviator isn't a good candidate for testing this particular feature :-)